Build failed because of webpack errors

node:16.4.0 
using the template with-typescript-tailwindcss
```code
Build error occurred
Error: > Build failed because of webpack errors
    at C:\Users\zhxie\Program files\Github\nuxt-electrion\node_modules\next\dist\build\index.js:15:924
    at async Span.traceAsyncFn (C:\Users\zhxie\Program files\Github\nuxt-electrion\node_modules\next\dist\telemetry\trace\trace.js:6:584)
```
after use 'yarn build'

I have experienced this too, right after upgrading to v7 of Nextron (Next v11.x).

Here is the full log I got out of building the project:

<details>
<summary>
⚠️ Don't expand if you're scared of long logs...
</summary>

```
$ nextron build
[nextron] Clearing previous builds
[nextron] Building renderer process
info  - Using webpack 5. Reason: Enabled by default nextjs.org/docs/messages/webpack5
warn  - No build cache found. Please configure build caching for faster rebuilds. Read more: nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
nextjs.org/telemetry

info  - Checking validity of types...
info  - Creating an optimized production build...
Failed to compile.

./styles/globals.css
HookWebpackError: __dirname is not defined
-- inner error --
ReferenceError: __dirname is not defined
Generated code for /home/runner/work/desktop/desktop/node_modules/next/dist/compiled/css-loader/api.js
1 | module.exports=function(){"use strict";var n={762:function(n){n.exports=function(n){var t=[];t.toString=function toString(){return this.map(function(t){var r=cssWithMappingToString(t,n);if(t[2]){return"@media ".concat(t[2]," {").concat(r,"}")}return r}).join("")};t.i=function(n,r,o){if(typeof n==="string"){n=[[null,n,""]]}var e={};if(o){for(var a=0;a<this.length;a++){var c=this[a][0];if(c!=null){e[c]=true}}}for(var i=0;i<n.length;i++){var u=[].concat(n[i]);if(o&&e[u[0]]){continue}if(r){if(!u[2]){u[2]=r}else{u[2]="".concat(r," and ").concat(u[2])}}t.push(u)}};return t};function cssWithMappingToString(n,t){var r=n[1]||"";var o=n[3];if(!o){return r}if(t&&typeof btoa==="function"){var e=toComment(o);var a=o.sources.map(function(n){return"/*# sourceURL=".concat(o.sourceRoot||"").concat(n," */")});return[r].concat(a).concat([e]).join("\n")}return[r].join("\n")}function toComment(n){var t=btoa(unescape(encodeURIComponent(JSON.stringify(n))));var r="sourceMappingURL=data:application/json;charset=utf-8;base64,".concat(t);return"/*# ".concat(r," */")}}};var t={};function __nccwpck_require__(r){if(t[r]){return t[r].exports}var o=t[r]={exports:{}};var e=true;try{n[r](o,o.exports,__nccwpck_require__);e=false}finally{if(e)delete t[r]}return o.exports}__nccwpck_require__.ab=__dirname+"/";return __nccwpck_require__(762)}();

Generated code for /home/runner/work/desktop/desktop/node_modules/next/dist/compiled/css-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[1]!/home/runner/work/desktop/desktop/node_modules/next/dist/compiled/postcss-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[2]!/home/runner/work/desktop/desktop/renderer/styles/globals.css
 1 | __webpack_require__.r(__webpack_exports__);
 2 | /* harmony import */ var _node_modules_next_dist_compiled_css_loader_api_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__("/home/runner/work/desktop/desktop/node_modules/next/dist/compiled/css-loader/api.js");
 3 | /* harmony import */ var _node_modules_next_dist_compiled_css_loader_api_js__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_node_modules_next_dist_compiled_css_loader_api_js__WEBPACK_IMPORTED_MODULE_0__);
 4 | // Imports
 5 | 
 6 | var ___CSS_LOADER_EXPORT___ = _node_modules_next_dist_compiled_css_loader_api_js__WEBPACK_IMPORTED_MODULE_0___default()(true);
 7 | ___CSS_LOADER_EXPORT___.push([module.id, "@import url(fonts.googleapis.com/css2?family=Lato&display=swap);"]);
 8 | // Module
 9 | ___CSS_LOADER_EXPORT___.push([module.id, "/*! tailwindcss v2.2.7 | MIT License | tailwindcss.com */\n\n/*! modern-normalize v1.1.0 | MIT License | sindresorhus/modern-normalize */\n\n/*\nDocument\n========\n*/\n\n/**\nUse a better box model (opinionated).\n*/\n\n*,\n::before,\n::after {\n  box-sizing: border-box;\n}\n\n/**\nUse a more readable tab size (opinionated).\n*/\n\nhtml {\n  -moz-tab-size: 4;\n  -o-tab-size: 4;\n     tab-size: 4;\n}\n\n/**\n1. Correct the line height in all browsers.\n2. Prevent adjustments of font size after orientation changes in iOS.\n*/\n\nhtml {\n  line-height: 1.15; /* 1 */\n  -webkit-text-size-adjust: 100%; /* 2 */\n}\n\n/*\nSections\n========\n*/\n\n/**\nRemove the margin in all browsers.\n*/\n\nbody {\n  margin: 0;\n}\n\n/**\nImprove consistency of default fonts in all browsers. (https://github.com/sindresorhus/modern-normalize/issues/3)\n*/\n\nbody {\n  font-family:\n\t\tsystem-ui,\n\t\t-apple-system, /* Firefox supports this but not yet `system-ui` */\n\t\t'Segoe UI',\n\t\tRoboto,\n\t\tHelvetica,\n\t\tArial,\n\t\tsans-serif,\n\t\t'Apple Color Emoji',\n\t\t'Segoe UI Emoji';\n}\n\n/*\nGrouping content\n================\n*/\n\n/**\n1. Add the correct height in Firefox.\n2. Correct the inheritance of border color in Firefox. (bugzilla.mozilla.org/show_bug.cgi?id=190655)\n*/\n\nhr {\n  height: 0; /* 1 */\n  color: inherit; /* 2 */\n}\n\n/*\nText-level semantics\n====================\n*/\n\n/**\nAdd the correct text decoration in Chrome, Edge, and Safari.\n*/\n\nabbr[title] {\n  -webkit-text-decoration: underline dotted;\n          text-decoration: underline dotted;\n}\n\n/**\nAdd the correct font weight in Edge and Safari.\n*/\n\nb,\nstrong {\n  font-weight: bolder;\n}\n\n/**\n1. Improve consistency of default fonts in all browsers. (https://github.com/sindresorhus/modern-normalize/issues/3)\n2. Correct the odd 'em' font sizing in all browsers.\n*/\n\ncode,\nkbd,\nsamp,\npre {\n  font-family:\n\t\tui-monospace,\n\t\tSFMono-Regular,\n\t\tConsolas,\n\t\t'Liberation Mono',\n\t\tMenlo,\n\t\tmonospace; /* 1 */\n  font-size: 1em; /* 2 */\n}\n\n/**\nAdd the correct font size in all browsers.\n*/\n\nsmall {\n  font-size: 80%;\n}\n\n/**\nPrevent 'sub' and 'sup' elements from affecting the line height in all browsers.\n*/\n\nsub,\nsup {\n  font-size: 75%;\n  line-height: 0;\n  position: relative;\n  vertical-align: baseline;\n}\n\nsub {\n  bottom: -0.25em;\n}\n\nsup {\n  top: -0.5em;\n}\n\n/*\nTabular data\n============\n*/\n\n/**\n1. Remove text indentation from table contents in Chrome and Safari. (bugs.chromium.org/p/chromium/issues/detail?id=999088, bugs.webkit.org/show_bug.cgi?id=201297)\n2. Correct table border color inheritance in all Chrome and Safari. (bugs.chromium.org/p/chromium/issues/detail?id=935729, bugs.webkit.org/show_bug.cgi?id=195016)\n*/\n\ntable {\n  text-indent: 0; /* 1 */\n  border-color: inherit; /* 2 */\n}\n\n/*\nForms\n=====\n*/\n\n/**\n1. Change the font styles in all browsers.\n2. Remove the margin in Firefox and Safari.\n*/\n\nbutton,\ninput,\noptgroup,\nselect,\ntextarea {\n  font-family: inherit; /* 1 */\n  font-size: 100%; /* 1 */\n  line-height: 1.15; /* 1 */\n  margin: 0; /* 2 */\n}\n\n/**\nRemove the inheritance of text transform in Edge and Firefox.\n1. Remove the inheritance of text transform in Firefox.\n*/\n\nbutton,\nselect { /* 1 */\n  text-transform: none;\n}\n\n/**\nCorrect the inability to style clickable types in iOS and Safari.\n*/\n\nbutton,\n[type='button'],\n[type='submit'] {\n  -webkit-appearance: button;\n}\n\n/**\nRemove the inner border and padding in Firefox.\n*/\n\n/**\nRestore the focus styles unset by the previous rule.\n*/\n\n/**\nRemove the additional ':invalid' styles in Firefox.\nSee: mozilla/gecko-dev@2f9eacd/layout/style/res/forms.css#L728-L737\n*/\n\n/**\nRemove the padding so developers are not caught out when they zero out 'fieldset' elements in all browsers.\n*/\n\nlegend {\n  padding: 0;\n}\n\n/**\nAdd the correct vertical alignment in Chrome and Firefox.\n*/\n\nprogress {\n  vertical-align: baseline;\n}\n\n/**\nCorrect the cursor style of increment and decrement buttons in Safari.\n*/\n\n/**\n1. Correct the odd appearance in Chrome and Safari.\n2. Correct the outline style in Safari.\n*/\n\n/**\nRemove the inner padding in Chrome and Safari on macOS.\n*/\n\n/**\n1. Correct the inability to style clickable types in iOS and Safari.\n2. Change font properties to 'inherit' in Safari.\n*/\n\n/*\nInteractive\n===========\n*/\n\n/*\nAdd the correct display in Chrome and Safari.\n*/\n\nsummary {\n  display: list-item;\n}\n\n/**\n * Manually forked from SUIT CSS Base: https://github.com/suitcss/base\n * A thin layer on top of normalize.css that provides a starting point more\n * suitable for web applications.\n */\n\n/**\n * Removes the default spacing and border for appropriate elements.\n */\n\nblockquote,\ndl,\ndd,\nh1,\nh2,\nh3,\nh4,\nh5,\nh6,\nhr,\nfigure,\np,\npre {\n  margin: 0;\n}\n\nbutton {\n  background-color: transparent;\n  background-image: none;\n}\n\nfieldset {\n  margin: 0;\n  padding: 0;\n}\n\nol,\nul {\n  list-style: none;\n  margin: 0;\n  padding: 0;\n}\n\n/**\n * Tailwind custom reset styles\n */\n\n/**\n * 1. Use the user's configured `sans` font-family (with Tailwind's default\n *    sans-serif font stack as a fallback) as a sane default.\n * 2. Use Tailwind's default \"normal\" line-height so the user isn't forced\n *    to override it to ensure consistency even when using the default theme.\n */\n\nhtml {\n  font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, \"Helvetica Neue\", Arial, \"Noto Sans\", sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\", \"Noto Color Emoji\"; /* 1 */\n  line-height: 1.5; /* 2 */\n}\n\n/**\n * Inherit font-family and line-height from `html` so users can set them as\n * a class directly on the `html` element.\n */\n\nbody {\n  font-family: inherit;\n  line-height: inherit;\n}\n\n/**\n * 1. Prevent padding and border from affecting element width.\n *\n *    We used to set this in the html element and inherit from\n *    the parent element for everything else. This caused issues\n *    in shadow-dom-enhanced elements like <details> where the content\n *    is wrapped by a div with box-sizing set to `content-box`.\n *\n *    https://github.com/mozdevs/cssremedy/issues/4\n *\n *\n * 2. Allow adding a border to an element by just adding a border-width.\n *\n *    By default, the way the browser specifies that an element should have no\n *    border is by setting it's border-style to `none` in the user-agent\n *    stylesheet.\n *\n *    In order to easily add borders to elements by just setting the `border-width`\n *    property, we change the default border-style for all elements to `solid`, and\n *    use border-width to hide them instead. This way our `border` utilities only\n *    need to set the `border-width` property instead of the entire `border`\n *    shorthand, making our border utilities much more straightforward to compose.\n *\n *    https://github.com/tailwindcss/tailwindcss/pull/116\n */\n\n*,\n::before,\n::after {\n  box-sizing: border-box; /* 1 */\n  border-width: 0; /* 2 */\n  border-style: solid; /* 2 */\n  border-color: currentColor; /* 2 */\n}\n\n/*\n * Ensure horizontal rules are visible by default\n */\n\nhr {\n  border-top-width: 1px;\n}\n\n/**\n * Undo the `border-style: none` reset that Normalize applies to images so that\n * our `border-{width}` utilities have the expected effect.\n *\n * The Normalize reset is unnecessary for us since we default the border-width\n * to 0 on all elements.\n *\n * https://github.com/tailwindcss/tailwindcss/issues/362\n */\n\nimg {\n  border-style: solid;\n}\n\ntextarea {\n  resize: vertical;\n}\n\ninput::-moz-placeholder, textarea::-moz-placeholder {\n  opacity: 1;\n  color: #9ca3af;\n}\n\ninput:-ms-input-placeholder, textarea:-ms-input-placeholder {\n  opacity: 1;\n  color: #9ca3af;\n}\n\ninput::placeholder,\ntextarea::placeholder {\n  opacity: 1;\n  color: #9ca3af;\n}\n\nbutton {\n  cursor: pointer;\n}\n\ntable {\n  border-collapse: collapse;\n}\n\nh1,\nh2,\nh3,\nh4,\nh5,\nh6 {\n  font-size: inherit;\n  font-weight: inherit;\n}\n\n/**\n * Reset links to optimize for opt-in styling instead of\n * opt-out.\n */\n\na {\n  color: inherit;\n  text-decoration: inherit;\n}\n\n/**\n * Reset form element properties that are easy to forget to\n * style explicitly so you don't inadvertently introduce\n * styles that deviate from your design system. These styles\n * supplement a partial reset that is already applied by\n * normalize.css.\n */\n\nbutton,\ninput,\noptgroup,\nselect,\ntextarea {\n  padding: 0;\n  line-height: inherit;\n  color: inherit;\n}\n\n/**\n * Use the configured 'mono' font family for elements that\n * are expected to be rendered with a monospace font, falling\n * back to the system monospace stack if there is no configured\n * 'mono' font family.\n */\n\npre,\ncode,\nkbd,\nsamp {\n  font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, \"Liberation Mono\", \"Courier New\", monospace;\n}\n\n/**\n * 1. Make replaced elements `display: block` by default as that's\n *    the behavior you want almost all of the time. Inspired by\n *    CSS Remedy, with `svg` added as well.\n *\n *    https://github.com/mozdevs/cssremedy/issues/14\n * \n * 2. Add `vertical-align: middle` to align replaced elements more\n *    sensibly by default when overriding `display` by adding a\n *    utility like `inline`.\n *\n *    This can trigger a poorly considered linting error in some\n *    tools but is included by design.\n * \n *    jensimmons/cssremedy/issues/14#issuecomment-634934210\n */\n\nimg,\nsvg,\nvideo,\ncanvas,\naudio,\niframe,\nembed,\nobject {\n  display: block; /* 1 */\n  vertical-align: middle; /* 2 */\n}\n\n/**\n * Constrain images and videos to the parent width and preserve\n * their intrinsic aspect ratio.\n *\n * https://github.com/mozdevs/cssremedy/issues/14\n */\n\nimg,\nvideo {\n  max-width: 100%;\n  height: auto;\n}\n\n/**\n * Ensure the default browser behavior of the `hidden` attribute.\n */\n\n*, ::before, ::after{\n  --tw-border-opacity: 1;\n  border-color: rgba(229, 231, 235, var(--tw-border-opacity));\n}\n\n.container{\n  width: 100%;\n}\n\n@media (min-width: 640px){\n  .container{\n    max-width: 640px;\n  }\n}\n\n@media (min-width: 768px){\n  .container{\n    max-width: 768px;\n  }\n}\n\n@media (min-width: 1024px){\n  .container{\n    max-width: 1024px;\n  }\n}\n\n@media (min-width: 1280px){\n  .container{\n    max-width: 1280px;\n  }\n}\n\n@media (min-width: 1536px){\n  .container{\n    max-width: 1536px;\n  }\n}\n\n.visible{\n  visibility: visible;\n}\n\n.fixed{\n  position: fixed;\n}\n\n.absolute{\n  position: absolute;\n}\n\n.top-2{\n  top: 0.5rem;\n}\n\n.mt-6{\n  margin-top: 1.5rem;\n}\n\n.mt-8{\n  margin-top: 2rem;\n}\n\n.mt-12{\n  margin-top: 3rem;\n}\n\n.mt-20{\n  margin-top: 5rem;\n}\n\n.mt-40{\n  margin-top: 10rem;\n}\n\n.mb-10{\n  margin-bottom: 2.5rem;\n}\n\n.mb-12{\n  margin-bottom: 3rem;\n}\n\n.flex{\n  display: flex;\n}\n\n.table{\n  display: table;\n}\n\n.h-6{\n  height: 1.5rem;\n}\n\n.h-screen{\n  height: 100vh;\n}\n\n.w-6{\n  width: 1.5rem;\n}\n\n.w-64{\n  width: 16rem;\n}\n\n.w-1\\/2{\n  width: 50%;\n}\n\n.w-full{\n  width: 100%;\n}\n\n.transform{\n  --tw-translate-x: 0;\n  --tw-translate-y: 0;\n  --tw-rotate: 0;\n  --tw-skew-x: 0;\n  --tw-skew-y: 0;\n  --tw-scale-x: 1;\n  --tw-scale-y: 1;\n  transform: translateX(var(--tw-translate-x)) translateY(var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y));\n}\n\n@-webkit-keyframes spin{\n  to{\n    transform: rotate(360deg);\n  }\n}\n\n@keyframes spin{\n  to{\n    transform: rotate(360deg);\n  }\n}\n\n@-webkit-keyframes ping{\n  75%, 100%{\n    transform: scale(2);\n    opacity: 0;\n  }\n}\n\n@keyframes ping{\n  75%, 100%{\n    transform: scale(2);\n    opacity: 0;\n  }\n}\n\n@-webkit-keyframes pulse{\n  50%{\n    opacity: .5;\n  }\n}\n\n@keyframes pulse{\n  50%{\n    opacity: .5;\n  }\n}\n\n@-webkit-keyframes bounce{\n  0%, 100%{\n    transform: translateY(-25%);\n    -webkit-animation-timing-function: cubic-bezier(0.8,0,1,1);\n            animation-timing-function: cubic-bezier(0.8,0,1,1);\n  }\n\n  50%{\n    transform: none;\n    -webkit-animation-timing-function: cubic-bezier(0,0,0.2,1);\n            animation-timing-function: cubic-bezier(0,0,0.2,1);\n  }\n}\n\n@keyframes bounce{\n  0%, 100%{\n    transform: translateY(-25%);\n    -webkit-animation-timing-function: cubic-bezier(0.8,0,1,1);\n            animation-timing-function: cubic-bezier(0.8,0,1,1);\n  }\n\n  50%{\n    transform: none;\n    -webkit-animation-timing-function: cubic-bezier(0,0,0.2,1);\n            animation-timing-function: cubic-bezier(0,0,0.2,1);\n  }\n}\n\n.cursor-pointer{\n  cursor: pointer;\n}\n\n.list-none{\n  list-style-type: none;\n}\n\n.flex-col{\n  flex-direction: column;\n}\n\n.content-center{\n  align-content: center;\n}\n\n.items-center{\n  align-items: center;\n}\n\n.justify-between{\n  justify-content: space-between;\n}\n\n.gap-5{\n  gap: 1.25rem;\n}\n\n.break-all{\n  word-break: break-all;\n}\n\n.rounded{\n  border-radius: 0.25rem;\n}\n\n.rounded-xl{\n  border-radius: 0.75rem;\n}\n\n.rounded-3xl{\n  border-radius: 1.5rem;\n}\n\n.border{\n  border-width: 1px;\n}\n\n.border-solid{\n  border-style: solid;\n}\n\n.border-none{\n  border-style: none;\n}\n\n.bg-white{\n  --tw-bg-opacity: 1;\n  background-color: rgba(255, 255, 255, var(--tw-bg-opacity));\n}\n\n.p-4{\n  padding: 1rem;\n}\n\n.px-4{\n  padding-left: 1rem;\n  padding-right: 1rem;\n}\n\n.px-5{\n  padding-left: 1.25rem;\n  padding-right: 1.25rem;\n}\n\n.py-4{\n  padding-top: 1rem;\n  padding-bottom: 1rem;\n}\n\n.text-center{\n  text-align: center;\n}\n\n.text-xl{\n  font-size: 1.25rem;\n  line-height: 1.75rem;\n}\n\n.text-2xl{\n  font-size: 1.5rem;\n  line-height: 2rem;\n}\n\n.text-4xl{\n  font-size: 2.25rem;\n  line-height: 2.5rem;\n}\n\n.text-5xl{\n  font-size: 3rem;\n  line-height: 1;\n}\n\n.text-6xl{\n  font-size: 3.75rem;\n  line-height: 1;\n}\n\n.text-black{\n  --tw-text-opacity: 1;\n  color: rgba(0, 0, 0, var(--tw-text-opacity));\n}\n\n.text-white{\n  --tw-text-opacity: 1;\n  color: rgba(255, 255, 255, var(--tw-text-opacity));\n}\n\n@media (prefers-color-scheme: dark){\n  .dark\\:text-white{\n    --tw-text-opacity: 1;\n    color: rgba(255, 255, 255, var(--tw-text-opacity));\n  }\n}\n\n.underline{\n  text-decoration: underline;\n}\n\n*, ::before, ::after{\n  --tw-shadow: 0 0 #0000;\n}\n\n.focus\\:outline-none:focus{\n  outline: 2px solid transparent;\n  outline-offset: 2px;\n}\n\n*, ::before, ::after{\n  --tw-ring-inset: var(--tw-empty,/*!*/ /*!*/);\n  --tw-ring-offset-width: 0px;\n  --tw-ring-offset-color: #fff;\n  --tw-ring-color: rgba(59, 130, 246, 0.5);\n  --tw-ring-offset-shadow: 0 0 #0000;\n  --tw-ring-shadow: 0 0 #0000;\n}\n\n.filter{\n  --tw-blur: var(--tw-empty,/*!*/ /*!*/);\n  --tw-brightness: var(--tw-empty,/*!*/ /*!*/);\n  --tw-contrast: var(--tw-empty,/*!*/ /*!*/);\n  --tw-grayscale: var(--tw-empty,/*!*/ /*!*/);\n  --tw-hue-rotate: var(--tw-empty,/*!*/ /*!*/);\n  --tw-invert: var(--tw-empty,/*!*/ /*!*/);\n  --tw-saturate: var(--tw-empty,/*!*/ /*!*/);\n  --tw-sepia: var(--tw-empty,/*!*/ /*!*/);\n  --tw-drop-shadow: var(--tw-empty,/*!*/ /*!*/);\n  filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow);\n}\n\n.transition{\n  transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow, transform, filter, -webkit-backdrop-filter;\n  transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow, transform, filter, backdrop-filter;\n  transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow, transform, filter, backdrop-filter, -webkit-backdrop-filter;\n  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);\n  transition-duration: 150ms;\n}\n\n.ease-out{\n  transition-timing-function: cubic-bezier(0, 0, 0.2, 1);\n}\n\n::-moz-selection {\n  color: black;\n  background: white;\n}\n\n::selection {\n  color: black;\n  background: white;\n}\n\ninput::-moz-selection {\n  color: white;\n  background: #157efb;\n}\n\ninput::selection {\n  color: white;\n  background: #157efb;\n}\n\n* {\n  font-family: \"Lato\", Helvetica, sans-serif;\n}\n\nbody {\n  background: #157efb;\n}\n\n.urlbar {\n  box-shadow: 0 4px 24px rgba(0, 0, 0, 0.25);\n  font-weight: 400;\n  padding: 0 10px;\n  height: 3.2rem;\n  border: 4px solid #fff;\n  width: 35rem;\n  max-width: 90vw;\n}\n\n.infor a {\n  text-decoration: underline;\n}\n\n.receive-bar {\n  font-size: 36px;\n}\n\n.receive-bar::-moz-placeholder {\n  filter: grayscale(100%);\n}\n\n.receive-bar:-ms-input-placeholder {\n  filter: grayscale(100%);\n}\n\n.receive-bar::placeholder {\n  filter: grayscale(100%);\n}\n\n@media (prefers-color-scheme: dark) {\n  body {\n    background: #151515;\n  }\n\n  .menu-item {\n    background: #262626;\n    color: white;\n  }\n}\n\n@media (min-width: 640px){\n}\n\n@media (min-width: 768px){\n}\n\n@media (min-width: 1024px){\n}\n\n@media (min-width: 1280px){\n}\n\n@media (min-width: 1536px){\n}\n", "",{"version":3,"sources":["webpack://styles/globals.css","<no source>"],"names":[],"mappings":"AAEA,gEAAc;;AAAd,8FAAc;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;;;EAAA,sBAAc;AAAA;;AAAd;;CAAc;;AAAd;EAAA,gBAAc;EAAd,cAAc;KAAd,WAAc;AAAA;;AAAd;;;CAAc;;AAAd;EAAA,iBAAc,EAAd,MAAc;EAAd,8BAAc,EAAd,MAAc;AAAA;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;EAAA,SAAc;AAAA;;AAAd;;CAAc;;AAAd;EAAA;;;;;;;;;kBAAc;AAAA;;AAAd;;;CAAc;;AAAd;;;CAAc;;AAAd;EAAA,SAAc,EAAd,MAAc;EAAd,cAAc,EAAd,MAAc;AAAA;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;EAAA,yCAAc;UAAd,iCAAc;AAAA;;AAAd;;CAAc;;AAAd;;EAAA,mBAAc;AAAA;;AAAd;;;CAAc;;AAAd;;;;EAAA;;;;;;WAAc,EAAd,MAAc;EAAd,cAAc,EAAd,MAAc;AAAA;;AAAd;;CAAc;;AAAd;EAAA,cAAc;AAAA;;AAAd;;CAAc;;AAAd;;EAAA,cAAc;EAAd,cAAc;EAAd,kBAAc;EAAd,wBAAc;AAAA;;AAAd;EAAA,eAAc;AAAA;;AAAd;EAAA,WAAc;AAAA;;AAAd;;;CAAc;;AAAd;;;CAAc;;AAAd;EAAA,cAAc,EAAd,MAAc;EAAd,qBAAc,EAAd,MAAc;AAAA;;AAAd;;;CAAc;;AAAd;;;CAAc;;AAAd;;;;;EAAA,oBAAc,EAAd,MAAc;EAAd,eAAc,EAAd,MAAc;EAAd,iBAAc,EAAd,MAAc;EAAd,SAAc,EAAd,MAAc;AAAA;;AAAd;;;CAAc;;AAAd;SAAA,MAAc;EAAd,oBAAc;AAAA;;AAAd;;CAAc;;AAAd;;;EAAA,0BAAc;AAAA;;AAAd;;CAAc;;AAAd;;CAAc;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;EAAA,UAAc;AAAA;;AAAd;;CAAc;;AAAd;EAAA,wBAAc;AAAA;;AAAd;;CAAc;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;;;CAAc;;AAAd;;;CAAc;;AAAd;;CAAc;;AAAd;EAAA,kBAAc;AAAA;;AAAd;;;;EAAc;;AAAd;;EAAc;;AAAd;;;;;;;;;;;;;EAAA,SAAc;AAAA;;AAAd;EAAA,6BAAc;EAAd,sBAAc;AAAA;;AAAd;EAAA,SAAc;EAAd,UAAc;AAAA;;AAAd;;EAAA,gBAAc;EAAd,SAAc;EAAd,UAAc;AAAA;;AAAd;;EAAc;;AAAd;;;;;EAAc;;AAAd;EAAA,4NAAc,EAAd,MAAc;EAAd,gBAAc,EAAd,MAAc;AAAA;;AAAd;;;EAAc;;AAAd;EAAA,oBAAc;EAAd,oBAAc;AAAA;;AAAd;;;;;;;;;;;;;;;;;;;;;;;;EAAc;;AAAd;;;EAAA,sBAAc,EAAd,MAAc;EAAd,eAAc,EAAd,MAAc;EAAd,mBAAc,EAAd,MAAc;EAAd,0BAAc,EAAd,MAAc;AAAA;;AAAd;;EAAc;;AAAd;EAAA,qBAAc;AAAA;;AAAd;;;;;;;;EAAc;;AAAd;EAAA,mBAAc;AAAA;;AAAd;EAAA,gBAAc;AAAA;;AAAd;EAAA,UAAc;EAAd,cAAc;AAAA;;AAAd;EAAA,UAAc;EAAd,cAAc;AAAA;;AAAd;;EAAA,UAAc;EAAd,cAAc;AAAA;;AAAd;EAAA,eAAc;AAAA;;AAAd;EAAA,yBAAc;AAAA;;AAAd;;;;;;EAAA,kBAAc;EAAd,oBAAc;AAAA;;AAAd;;;EAAc;;AAAd;EAAA,cAAc;EAAd,wBAAc;AAAA;;AAAd;;;;;;EAAc;;AAAd;;;;;EAAA,UAAc;EAAd,oBAAc;EAAd,cAAc;AAAA;;AAAd;;;;;EAAc;;AAAd;;;;EAAA,+GAAc;AAAA;;AAAd;;;;;;;;;;;;;;;EAAc;;AAAd;;;;;;;;EAAA,cAAc,EAAd,MAAc;EAAd,sBAAc,EAAd,MAAc;AAAA;;AAAd;;;;;EAAc;;AAAd;;EAAA,eAAc;EAAd,YAAc;AAAA;;AAAd;;EAAc;;AAAd;EAAA,sBAAc;EAAd,2DAAA;AAAc;;AACd;EAAA,WAAA;AAAoB;;AAApB;EAAA;IAAA,gBAAA;EAAoB;AAAA;;AAApB;EAAA;IAAA,gBAAA;EAAoB;AAAA;;AAApB;EAAA;IAAA,iBAAA;EAAoB;AAAA;;AAApB;EAAA;IAAA,iBAAA;EAAoB;AAAA;;AAApB;EAAA;IAAA,iBAAA;EAAoB;AAAA;;AACpB;EAAA,mBAAA;AAAmB;;AAAnB;EAAA,eAAA;AAAmB;;AAAnB;EAAA,kBAAA;AAAmB;;AAAnB;EAAA,WAAA;AAAmB;;AAAnB;EAAA,kBAAA;AAAmB;;AAAnB;EAAA,gBAAA;AAAmB;;AAAnB;EAAA,gBAAA;AAAmB;;AAAnB;EAAA,gBAAA;AAAmB;;AAAnB;EAAA,iBAAA;AAAmB;;AAAnB;EAAA,qBAAA;AAAmB;;AAAnB;EAAA,mBAAA;AAAmB;;AAAnB;EAAA,aAAA;AAAmB;;AAAnB;EAAA,cAAA;AAAmB;;AAAnB;EAAA,cAAA;AAAmB;;AAAnB;EAAA,aAAA;AAAmB;;AAAnB;EAAA,aAAA;AAAmB;;AAAnB;EAAA,YAAA;AAAmB;;AAAnB;EAAA,UAAA;AAAmB;;AAAnB;EAAA,WAAA;AAAmB;;AAAnB;EAAA,mBAAmB;EAAnB,mBAAmB;EAAnB,cAAmB;EAAnB,cAAmB;EAAnB,cAAmB;EAAnB,eAAmB;EAAnB,eAAmB;EAAnB,2MAAA;AAAmB;;AAAnB;EAAA;IAAA,yBAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,yBAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,mBAAmB;IAAnB,UAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,mBAAmB;IAAnB,UAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,WAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,WAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,2BAAmB;IAAnB,0DAAmB;YAAnB,kDAAA;EAAmB;;EAAnB;IAAA,eAAmB;IAAnB,0DAAmB;YAAnB,kDAAA;EAAmB;AAAA;;AAAnB;EAAA;IAAA,2BAAmB;IAAnB,0DAAmB;YAAnB,kDAAA;EAAmB;;EAAnB;IAAA,eAAmB;IAAnB,0DAAmB;YAAnB,kDAAA;EAAmB;AAAA;;AAAnB;EAAA,eAAA;AAAmB;;AAAnB;EAAA,qBAAA;AAAmB;;AAAnB;EAAA,sBAAA;AAAmB;;AAAnB;EAAA,qBAAA;AAAmB;;AAAnB;EAAA,mBAAA;AAAmB;;AAAnB;EAAA,8BAAA;AAAmB;;AAAnB;EAAA,YAAA;AAAmB;;AAAnB;EAAA,qBAAA;AAAmB;;AAAnB;EAAA,sBAAA;AAAmB;;AAAnB;EAAA,sBAAA;AAAmB;;AAAnB;EAAA,qBAAA;AAAmB;;AAAnB;EAAA,iBAAA;AAAmB;;AAAnB;EAAA,mBAAA;AAAmB;;AAAnB;EAAA,kBAAA;AAAmB;;AAAnB;EAAA,kBAAmB;EAAnB,2DAAA;AAAmB;;AAAnB;EAAA,aAAA;AAAmB;;AAAnB;EAAA,kBAAmB;EAAnB,mBAAA;AAAmB;;AAAnB;EAAA,qBAAmB;EAAnB,sBAAA;AAAmB;;AAAnB;EAAA,iBAAmB;EAAnB,oBAAA;AAAmB;;AAAnB;EAAA,kBAAA;AAAmB;;AAAnB;EAAA,kBAAmB;EAAnB,oBAAA;AAAmB;;AAAnB;EAAA,iBAAmB;EAAnB,iBAAA;AAAmB;;AAAnB;EAAA,kBAAmB;EAAnB,mBAAA;AAAmB;;AAAnB;EAAA,eAAmB;EAAnB,cAAA;AAAmB;;AAAnB;EAAA,kBAAmB;EAAnB,cAAA;AAAmB;;AAAnB;EAAA,oBAAmB;EAAnB,4CAAA;AAAmB;;AAAnB;EAAA,oBAAmB;EAAnB,kDAAA;AAAmB;;ACJnB;EDIA;IAAA,oBAAmB;IAAnB,kDAAA;EAAmB;AAo0BnB;;AAp0BA;EAAA,0BAAA;AAAmB;;AAAnB;EAAA,sBAAA;AAAmB;;AAAnB;EAAA,8BAAmB;EAAnB,mBAAA;AAAmB;;AAAnB;EAAA,4CAAmB;EAAnB,2BAAmB;EAAnB,4BAAmB;EAAnB,wCAAmB;EAAnB,kCAAmB;EAAnB,2BAAA;AAAmB;;AAAnB;EAAA,sCAAmB;EAAnB,4CAAmB;EAAnB,0CAAmB;EAAnB,2CAAmB;EAAnB,4CAAmB;EAAnB,wCAAmB;EAAnB,0CAAmB;EAAnB,uCAAmB;EAAnB,6CAAmB;EAAnB,iLAAA;AAAmB;;AAAnB;EAAA,yIAAmB;EAAnB,iIAAmB;EAAnB,0JAAmB;EAAnB,wDAAmB;EAAnB,0BAAA;AAAmB;;AAAnB;EAAA,sDAAA;AAAmB;;AAEnB;EACE,YAAY;EACZ,iBAAiB;AACnB;;AAHA;EACE,YAAY;EACZ,iBAAiB;AACnB;;AAEA;EACE,YAAY;EACZ,mBAAmB;AACrB;;AAHA;EACE,YAAY;EACZ,mBAAmB;AACrB;;AAEA;EACE,0CAA0C;AAC5C;;AAEA;EACE,mBAAmB;AACrB;;AAEA;EACE,0CAA0C;EAC1C,gBAAgB;EAChB,eAAe;EACf,cAAc;EACd,sBAAsB;EAEtB,YAAY;EACZ,eAAe;AACjB;;AAEA;EACE,0BAA0B;AAC5B;;AAEA;EACE,eAAe;AACjB;;AAEA;EACE,uBAAuB;AACzB;;AAFA;EACE,uBAAuB;AACzB;;AAFA;EACE,uBAAuB;AACzB;;AAEA;EACE;IACE,mBAAmB;EACrB;;EACA;IACE,mBAAmB;IACnB,YAAY;EACd;AACF;;ACvDA;AD+7BA;;AC/7BA;ADk8BA;;ACl8BA;ADq8BA;;ACr8BA;ADw8BA;;ACx8BA;AD28BA","sourcesContent":["@import url(\"fonts.googleapis.com/css2?family=Lato&display=swap\");\n\n@tailwind base;\n@tailwind components;\n@tailwind utilities;\n\n::selection {\n  color: black;\n  background: white;\n}\n\ninput::selection {\n  color: white;\n  background: #157efb;\n}\n\n* {\n  font-family: \"Lato\", Helvetica, sans-serif;\n}\n\nbody {\n  background: #157efb;\n}\n\n.urlbar {\n  box-shadow: 0 4px 24px rgba(0, 0, 0, 0.25);\n  font-weight: 400;\n  padding: 0 10px;\n  height: 3.2rem;\n  border: 4px solid #fff;\n\n  width: 35rem;\n  max-width: 90vw;\n}\n\n.infor a {\n  text-decoration: underline;\n}\n\n.receive-bar {\n  font-size: 36px;\n}\n\n.receive-bar::placeholder {\n  filter: grayscale(100%);\n}\n\n@media (prefers-color-scheme: dark) {\n  body {\n    background: #151515;\n  }\n  .menu-item {\n    background: #262626;\n    color: white;\n  }\n}\n",null],"sourceRoot":""}]);
10 | // Exports
11 | /* harmony default export */ __webpack_exports__["default"] = (___CSS_LOADER_EXPORT___);
12 |


> Build error occurred
Error: > Build failed because of webpack errors
    at /home/runner/work/desktop/desktop/node_modules/next/dist/build/index.js:15:924
    at async Span.traceAsyncFn (/home/runner/work/desktop/desktop/node_modules/next/dist/telemetry/trace/trace.js:6:584)

error Command failed with exit code 1.

Cannot build electron packages:
Error: Command failed with exit code 1: next build /home/runner/work/desktop/desktop/renderer

info Visit yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

</details>

--- a/examples/with-typescript-tailwindcss/renderer/next.config.js
+++ b/examples/with-typescript-tailwindcss/renderer/next.config.js
@@ -2,6 +2,9 @@ module.exports = {
   webpack: (config, { isServer }) => {
     if (!isServer) {
       config.target = 'electron-renderer';
+      config.node = {
+        __dirname: true,
+      }
     }
 
     return config;

